### DISCUSSION TITLE
Inserting images in the context page: error message

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#674] [General] [answered]
When I try to insert an image anywhere in the context page I get the following error message:

![afbeelding](https://user-images.githubusercontent.com/102361398/174317290-09fc44ee-274a-44cb-a2f4-2bcfc217deaa.png)

**Comment by techsmyth** [comment]
Thanks Gerlof for reporting this. 

@Comoque1 Can you please reproduce this as it may well be caused by the recent data validation work? Thanks, Neil

**Comment by Comoque1** [comment, marked as answer]
> When I try to insert an image anywhere in the context page I get the following error message:
> 
> ![afbeelding](https://user-images.githubusercontent.com/102361398/174317290-09fc44ee-274a-44cb-a2f4-2bcfc217deaa.png)

Hi Gerlof,

Could you please, share details, like: image source and context (the Hub / Challenge / Opportunity ) name?
I did double check on develop environment, and wasn't able to reproduce the issue.

Thanks,
Evgeni

**Reply by GerlofdeHaan** [reply]
Hi Evgeni,

I'm working on: https://hub.alkem.io/digileefomgeving/challenges/t3d/dashboard
And I was trying to insert an image on the context page: https://hub.alkem.io/digileefomgeving/challenges/t3d/context

The image source: https://gemeentelijkgeoberaad.pleio.nl/file/download/e62f9c7e-01b3-40f6-8317-bfbb8a11d27a/figuur-1-het-programma-in-fasen.jpg

Gr. Gerlof

**Reply by Comoque1** [reply]
> https://hub.alkem.io/digileefomgeving/challenges/t3d/dashboard

Thank you for sharing this details, Gerlof!
To which field, would you like the image to be applied, i.e.: `Vision`, `Background`, `Impact`, `Who`?
I just tried setting the image up for field `Vision` and it is visible in the preview:
![image](https://user-images.githubusercontent.com/23620658/175898626-76ad5aa5-33a2-49ed-847d-dadad06c6fb7.png)

Please, let me know, if you would like to make the change for you. Just confirm the field.

Regards,
Evgeni

**Reply by GerlofdeHaan** [reply]
The Vision field is ok. I managed to get this far as well. But when I try to save it as an update I get the error message.
So try to save it now and see if it works. If so, then thats great offcourse.

Gr Gerlof

**Reply by Comoque1** [reply]
> The Vision field is ok. I managed to get this far as well. But when I try to save it as an update I get the error message. So try to save it now and see if it works. If so, then thats great offcourse.
> 
> Gr Gerlof

I see now, the issue comes from validation on `Background` field length. The challenge cannot be updated, due to this error, hence the change with the image cannot be saved.

I raised high severity bug in our backlog.

Regards,
Evgeni

**Reply by Comoque1** [reply]
Hi Gerlof,

I am writing to inform you, the the bug mentioned above is fixed.
I also uploaded the image to context / vision block.

![image](https://user-images.githubusercontent.com/23620658/176424036-ffe3d091-3e3f-450b-9927-f90753984248.png)


If you have any other concerns or issues, please let us know.

Regards,
Evgeni

**Reply by GerlofdeHaan** [reply]
Nice! thanks.

Gr. Gerlof
